packages/core/src/reports/utils.ts

```
@@ -498,3 +330,1587 @@
+  if (typeof bs === 'boolean') {
+    return 1;
+  }
+
+  if (!as || as.trim() === "") {
+    return 1;
+  }
+  if (!bs || bs.trim() === "") {
+    return -1;
+  }
+
+  if (typeof as === 'number' && isNaN(as)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)